src/opnsense/www/js/widgets/Certificates.js

```diff
@@ -73,7 +73,7 @@ export default class Certificates extends BaseTableWidget {
 
     processItems(items, type, hiddenItems, rows) {
         items.forEach(item => {
-            if (!hiddenItems.includes(item.descr)) {
+            if (!hiddenItems.includes(item.uuid)) {
```

Comment:
hiddenItems is set elsewhere in the code as `config.hiddenItems || []`. Best use `getWidgetConfig().hiddenItems` instead so the default is set based on what's returned by `setWidgetConfig()`. In this case the result is still an empty array, but at least it adheres to the proper logic.